molo/profiles/wagtail_hooks.py

```diff
@@ -0,0 +1,71 @@
+from daterange_filter.filter import DateRangeFilter
+from django.contrib.auth.models import User
+from django.http import HttpResponse
+from molo.profiles.admin import ProfileUserAdmin
+from import_export import resources
+from wagtailmodeladmin.options import ModelAdmin, wagtailmodeladmin_register
+from wagtailmodeladmin.views import IndexView
+
+
+class DateFilter(DateRangeFilter):
+    template = 'admin/user_date_range_filter.html'
+
+
+class FrontendUsersResource(resources.ModelResource):
+    class Meta:
+        model = User
+        exclude = ('id', 'password', 'is_superuser', 'groups',
+                   'user_permissions', 'is_staff')
+
+
+class ModelAdminTemplate(IndexView):
```

Comment:
Please rename to FrontendUsersAdminView, rename the associated template and move to admin_views.py